dali/python/nvidia/dali/external_source.py

```diff
@@ -615,6 +615,7 @@ def __call__(
                         op_instance._layout = layout
                 else:
                     op_instance._layout = None
+                op_instance._batch = batch
```

Comment:
just  thought: the word "batch" on its own makes me think that this is the actual batch, while it is probably a flag telling whether this is an instance providing batches. Consider `is_batched` or similar to avoid ambiguity

Reply:
Update: maybe `batched`

Reply:
In the context of external source `batch` means the batch parameter. I know the name is not ideal, but all the internal flags match the original paremeters, like `_source`, `_layout` etc. 
I don't want to start tinkering with this logic, and suddenly have 
```
layout = self._layout
cuda_stream = self._cuda_stream
batch = self._is_batch_mode
```